pkg/config/constants.go

```diff
@@ -92,7 +92,7 @@ const (
 	AllowedPeer = "allowedPeer"
 	Signature   = "signature"
 	Debug       = "debug"
-	Version     = "v0.0.4-beta"
+	Version     = "v0.0.5-beta"
```

Comment:
small nit for the future as I see this information already scattered across multiple places: we should really look into grabbing the version during build time rather then managing it in the code